### DISCUSSION TITLE
Error in Github Actions integration "affected stacks" docs

### DISCUSSION THREAD

**Discussion by rwohleb** [cloudposse/community#18] [Free Tier Support] [answered]
### Describe the Bug

The usage example defines two jobs: `context` and `atmos-plan`. The `atmos-plan` job sets a "needs" dependency to `atmos-affected` which doesn't exist.

### Expected Behavior

Either the `context` job needs to be renamed to `atmos-affected`, or the "needs" parameter needs to be updated to reference the `context` job.

### Steps to Reproduce

None.

**Comment by milldr** [comment]
Can you please link the specific docs page? We have these actions documented in a few places

**Reply by milldr** [reply]
https://atmos.tools/integrations/github-actions/affected-stacks

Was it this one? I believe I found what you're referring to. Thanks for pointing it out!

**Reply by rwohleb** [reply]
Yep, that's the one.

**Reply by milldr** [reply, marked as answer]
For reference - https://github.com/cloudposse/atmos/pull/661

**Reply by milldr** [reply]
Updated now! Please let me know if that solved the issue for you

**Reply by rwohleb** [reply]
I haven't verified the change, but that looks good to me. Thanks!

**Reply by rwohleb** [reply]
I'm assuming the update to the https://github.com/cloudposse/github-action-atmos-affected-stacks README will come later?

**Reply by milldr** [reply]
Yup with this PR: https://github.com/cloudposse/github-action-atmos-affected-stacks/pull/44
